lava-frappe-sdk/README.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
The readme needs to be updated with an overview of the project, as well as the steps necessary to work on it if any.